scripts/daobet_build.sh

```diff
@@ -284,11 +288,14 @@ printf "## ENABLE_COVERAGE_TESTING=%s\\n" "${ENABLE_COVERAGE_TESTING}"
 mkdir -p $BUILD_DIR
 cd $BUILD_DIR
 
+export BOOST_ROOT=${OPT_LOCATION}/boost
```

Comment:
It overrides the assignment in the beginning of the file.

Reply:
fixed